runhouse/logger.py

```diff
@@ -90,6 +91,51 @@ def formatTime(self, record, datefmt=None):
             return dt.isoformat(timespec="milliseconds")
 
 
+def get_logger(name: str = __name__, log_level: Union[int, str] = logging.INFO):
+    level_name = (
+        logging.getLevelName(log_level) if isinstance(log_level, int) else log_level
+    )
+    LOGGING_CONFIG = {
```

Comment:
this is a bit redundant, since we are defining the config here in this function and also below as part of the default logger.

instead of having two paths to import the logger:
(1) Default
```
import logging
logger = logging.getLogger(__name__)
```
(2) Customized Log Level
```
from runhouse.logger import get_logger
logger = get_logger(name=__name__, log_level=DEFAULT_LOG_LEVEL)
```

maybe we should have everything flow thru the second path. wdyt?